tst/EnergyPlus/unit/MixedAir.unit.cc

```diff
@@ -65,6 +65,7 @@
 #include <EnergyPlus/MixedAir.hh>
 #include <EnergyPlus/DataContaminantBalance.hh>
 #include <EnergyPlus/DataAirLoop.hh>
+#include <EnergyPlus/DataAirSystems.hh>
```

Comment:
Looks like the clear states are already done for these new includes, so that's good.